src/applications/appeals/996/config/form.js

```diff
@@ -37,7 +37,7 @@ import { CONTESTABLE_ISSUES_PATH } from '../../shared/constants';
 import { appStateSelector } from '../../shared/utils/issues';
 import reviewErrors from '../../shared/content/reviewErrors';
 
-// import initialData from '../tests/schema/initialData';
+// import initialData from '../tests/initialData';
```

Comment:
Are we keeping because we might we use later?

Reply:
Hmmm, I guess? InitialData is kinda like prefiling the form, or using the save-in-progress menu. I didn't see a reason to delete it, but I guess we can look into it.